app/Http/Controllers/JobController.php

```diff
@@ -57,21 +57,20 @@ public function managerIndex()
         $manager = Auth::user()->manager;
 
         $jobs = JobPoster::where('manager_id', $manager->id)
+            ->with('classification')
```

Comment:
This was the key piece missing to actually fix the issue I reported 😆

Reply:
😢. This is the gold right here eh: https://laravel.com/docs/5.8/eloquent-relationships#eager-loading